iothub/device/src/Transport/RetryDelegatingHandler.cs

```diff
@@ -588,7 +589,7 @@ private async Task HandleDisconnect()
 
                 // We were not able to recover the connection or subscriptions within the configured policy.
                 // The object will be placed in an unusable state.
-                Dispose(true);
+                // Dispose(true);
```

Comment:
Commented code should be removed. Since you removed this, you'll need to change the comment explaining how this is going to be handled now.

Reply:
removed